lang/en.json

```diff
@@ -67,6 +67,7 @@
         },
         "label": "Your Cart ({quantity, plural, one {# item} other {# items}})",
         "is_empty": "Your cart is empty",
+        "invalid_entry_message": "{entry} is not a valid entry",
```

Comment:
Could we custom placeholder here instead of undefined?

Reply:
I believe `undefined` will suite us better:
-  when we take this localized string from context a value already is set to `undefined` so it's safe to find it in the string and replace.
- using a custom placeholder we will add a risk that it can be accidentally translated later on. In this case we won't be able to find it & replace in js. 

If you still think a custom placeholder is better, I will change it.

Reply:
<img width="868" alt="Screenshot 2021-05-19 at 13 45 57" src="https://user-images.githubusercontent.com/66319629/118800442-947a6d80-b8a8-11eb-82a2-0afbcea119a6.png">

Reply:
yeah, I know:) we've got  a few of them but I still think they are little bit different cases. 
- firstly, we didn't have a choice there. Depends on language specifics SLIDE_NUMBER or SLIDES_QUANTITY can be placed in string in different order. As a result we can have 2 `undefined` and which of them replace with which value is a question.
- secondly, we've got a little trouble with custom placeholders so we even created a ticket [STRF-8764](https://jira.bigcommerce.com/browse/STRF-8764) to modify lang helper. I'm not sure it will be taken in work or not. Some quotes from there:
After discussion with Localization Team  we found a few downsides:

  - Adding a new kind of brackets will lead to explaining these use cases for Merchants and Dev community. When they should use square brackets and when should not. It can add more complexity for them and for us in a perspective since we need to keep this brackets as they are in en.json without translating.
  
  - New brackets will result in changing documentation (in more extended way comparing to updating lang helper).
  
still change to a placeholder?

Reply:
I believe we should follow the consistent approach, and if we have mechanism for similar task we could use it. In case if you want to change it, please change all square brackets logic